.github/workflows/build.yml

```diff
@@ -49,4 +49,17 @@ jobs:
           platforms: linux/amd64,linux/arm64,linux/arm/v7
           push: true
           tags: ${{ steps.meta_agent.outputs.tags }}
-          labels: ${{ steps.meta_agent.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.meta_agent.outputs.labels }}
+
+      - name: Run Trivy vulnerability scanner
```

Comment:
This step runs on master branch silently and then uploads the results to the "Security" tab